scripts/wallet_web_start.sh

```diff
@@ -8,5 +8,11 @@ set -e
 
 echo "Running $0"
 
+# Use corresponding image based on architecture type (AMD/ARM)
+arch=$(uname -p)
+if  [[ $arch == arm* ]]; then
```

Comment:
I think we should check for an exact architecture match since there are technically different `arm` architectures, and the MySQL build below is for `arm64`